samples/topology/src/main/java/quarks/samples/topology/CombiningStreamsProcessingResults.java

```diff
@@ -0,0 +1,158 @@
+/*
```

Comment:
These are new files you are contributing to the ASF under the ICLA, thus they only have the ASF licence header.